dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/MapUtils.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.common.utils;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class MapUtils {
+
+    public static void putAll(Map<String, List<String>> originMap,
+                                  Map<String, List<String>> newMap) {
+        for (String key : newMap.keySet()) {
+            List<String> filePaths = originMap.get(key);
+            if (filePaths == null) {
+                filePaths = new ArrayList<>();
+                originMap.put(key, filePaths);
+            }
+            filePaths.addAll(newMap.get(key));
+        }
+    }
+}
```

Comment:
This is an overuse of util class in my opinion, it's just a one-line codes and not a common case, I don't know who else will use this util class because it seems that this function is only used in your case.

```java
newMap.forEach((key, val) -> originMap.computeIfAbsent(key, __ -> new ArrayList<>()).addAll(val));
```